### ISSUE TITLE
How to install python packages?

### ISSUE THREAD

**Issue by rodrigjg** [lambci/docker-lambda#57]
Hi I was very impressed with your work here so helpful! I was wondering how I would go about adding pip packages to these docker containers. I can't seem to find documentation on it anywhere. I am using this package as part of the serverless-plugin-simulate plugin. I was also wondering what I would have to do in order to make this jive well with the serverless-python-requirements plugin. Thanks!

**Comment by jamespegg** [comment]
@rodrigjg Did you manage to figure how to do it?

**Comment by rodrigjg** [comment]
@jamespegg Yes! I used this along with the serverless simulate plugin. I looked at what made the run and build images they have different and used the code from [serverless-python-requirements](https://github.com/UnitedIncome/serverless-python-requirements) to figure out what to do. Eventually I came up with this docker file which pulls from the build image. 

```
FROM lambci/lambda:build-python3.6

RUN pip3 install elasticsearch

RUN rm /var/runtime/awslambda/runtime.cpython-36m-x86_64-linux-gnu.so
COPY runtime-mock.py /var/runtime/awslambda/runtime.py

#USER sbx_user1051

ENTRYPOINT ["/var/lang/bin/python3.6", "/var/runtime/awslambda/bootstrap.py"]
```

You will have to get the runtime.py file from [here](https://github.com/lambci/docker-lambda/tree/master/python3.6/run)

With this file in your current directory and same with the Dockerfile above you can docker build give the new docker image a name and tada you will have a docker image. To user this with serverless simulate specify the name of the new image as the run time for the function.

**Comment by jamespegg** [comment]
@rodrigjg Awesome, that worked! I had to prepend the Docker tag with `lambci/lambda:` and then it worked.

Have you thought about suggesting a documentation change to the serverless simulate plugin to cover this use case? It really helped me out and I'm sure we're not the only ones with this problem. :)

**Comment by mrpatrick** [comment]
I know this is closed but thought I'd share - found this tool that does the "packaging" for you so you don't have to build your own docker file and docker-based (no need for node).  Just reads the requirements file and puts everything in your working dir.  I found this to be a little easier to insert into my workflow:

https://github.com/myrmex-org/docker-lambda-packager

**Comment by mhart** [comment]
@mrpatrick there's no need for node – that's only required if you want to use the test runner.

**Comment by brq-cr** [comment]
Hey, I went able to make it work with the answer above. However I had to spend some time to understand what are those steps to create the docker image because I didn't had nice docker knowledge.
So if somebody comes here with the same problem just run the following command in your root directory to build the docker image:

`build . -t lambci/lambda:MY-CUSTOM-TAG`

Where "MY-CUSTOM-TAG" is the custom tag name I should use in the runtime. Then just update the runtime configuration in the serverless.yml:

`provider:
  name: aws	  name: aws
  runtime: python3.6	  runtime: MY-CUSTOM-TAG`
